setup.py

```diff
@@ -38,6 +38,18 @@
           long_description=open('README.md').read(),
           long_description_content_type='text/markdown',
           python_requires='~=3.6',
+          install_requires=[
+              'mne >=0.21',
+              'numpy >=1.14',
+              'scipy >=0.18.1',
+          ],
+          extras_require={
+              'full': [
+                  'nibabel >=2.2',
+                  'pybv >=0.4',
+                  'matplotlib'
```

Comment:
why is pandas not listed here, but it's listed as an optional dependency in the installation instructions?

Reply:
Good question! I will add it.